docs/instrumentation-list.yaml

```diff
@@ -0,0 +1,1603 @@
+activej:
+  instrumentations:
+  - name: activej-http-6.0
+    srcPath: instrumentation/activej-http-6.0
+    target_versions:
+      JAVAAGENT:
+      - io.activej:activej-http:[6.0,)
+akka:
+  instrumentations:
+  - name: akka-http-10.0
+    srcPath: instrumentation/akka/akka-http-10.0
+    target_versions:
+      JAVAAGENT:
+      - com.typesafe.akka:akka-http_2.12:[10,)
+      - com.typesafe.akka:akka-http_2.13:[10,)
+      - com.typesafe.akka:akka-http_2.11:[10,)
+  - name: akka-actor-fork-join-2.5
+    srcPath: instrumentation/akka/akka-actor-fork-join-2.5
+    target_versions:
+      JAVAAGENT:
+      - com.typesafe.akka:akka-actor_2.12:[2.5,2.6)
+      - com.typesafe.akka:akka-actor_2.13:[2.5.23,2.6)
+      - com.typesafe.akka:akka-actor_2.11:[2.5,)
+  - name: akka-actor-2.3
+    srcPath: instrumentation/akka/akka-actor-2.3
+    target_versions:
+      JAVAAGENT:
+      - com.typesafe.akka:akka-actor_2.11:[2.3,)
+      - com.typesafe.akka:akka-actor_2.12:[2.3,)
+      - com.typesafe.akka:akka-actor_2.13:[2.3,)
+alibaba:
+  instrumentations:
+  - name: alibaba-druid-1.0
+    srcPath: instrumentation/alibaba-druid-1.0
+    target_versions:
+      JAVAAGENT:
+      - com.alibaba:druid:(,)
+      LIBRARY:
+      - com.alibaba:druid:1.0.0
+apache:
+  instrumentations:
+  - name: apache-shenyu-2.4
+    srcPath: instrumentation/apache-shenyu-2.4
+    target_versions:
+      JAVAAGENT:
+      - org.apache.shenyu:shenyu-web:[2.4.0,)
+  - name: apache-httpclient-2.0
+    srcPath: instrumentation/apache-httpclient/apache-httpclient-2.0
+    target_versions:
+      JAVAAGENT:
+      - commons-httpclient:commons-httpclient:[2.0,4.0)
+  - name: apache-httpasyncclient-4.1
+    srcPath: instrumentation/apache-httpasyncclient-4.1
+    target_versions:
+      JAVAAGENT:
+      - org.apache.httpcomponents:httpasyncclient:[4.1,)
+  - name: apache-httpclient-4.3
+    srcPath: instrumentation/apache-httpclient/apache-httpclient-4.3
+    target_versions:
+      LIBRARY:
+      - org.apache.httpcomponents:httpclient:4.3
```

Comment:
we could potentially find the upperbound via the `latestDepTestLibrary` declaration